OptimizelySDKShared/OptimizelySDKShared/OPTLYManagerBase.m

```diff
@@ -67,6 +69,29 @@ @interface OPTLYManagerBase()
 
 @implementation OPTLYManagerBase
 
+- (void)cleanUserProfileService:(NSArray<OPTLYExperiment> *)experiments {
+    if (experiments == nil) return;
+    
+    if (_userProfileService != nil && [(NSObject *)_userProfileService respondsToSelector:@selector(removeInvalidExperimentsForAllUsers:)]) {
```

Comment:
This seems dangerous, as in, should we wrap it in a `try/catch`?

Reply:
respondsToSelector is a test.  I can't think of a possible reason an exception could be thrown here.